content/2017-09-19_pycon-cz-conference-wrap-up-summary.md

```diff
@@ -0,0 +1,199 @@
+
+We did not manage to have dedicated mobile apps for the event as last year. On the other hand, we had a UX-aware DTP-capable frontend designer as part of the core team, which resulted in many improvements in presentation of the conference, especially the usability of the website from mobile devices.
+
+## Content Stats
+
+- 31 talks (4 keynotes), 30 speakers
+- 9 workshops
+
+## Ticket Stats
+
+- 361 tickets, 334 attendees (20% women)
+- 62% of tickets were paid
+    - 24% (39% of all paid) **Self-funded** (38% of them were Early Bird)
+    - 22% (35% of all paid) **Employer-funded** (40% of them were Early Bird)
+    - 15% (24% of all paid) **Student/Teacher** (49% of them were Early Bird)
+    - 3 tickets for PyLadies
+- 38% of tickets were free
+    - 11% sponsors
+    - 11% speakers and workshop leaders
+    - 7% Django Girls (from that 35% were coaches)
+    - 4% Organizers Team PyCon CZ
+    - 4 tickets given away in raffles at meetups
+    - 4 tickets Financial Aid
+    - 3 tickets GymGeek
+    - 4 tickets PyCon PL/SK organizers
+- 27 people did not come (4 Django Girls, 3 Django Girls coaches, 6 Early Bird student/teachers, 2 student/teachers, 5 self-funded, 3 employer-funded, 3 sponsors, 1 PyCon SK organizer)
+
+## Feedback
+
+We've been running a feedback survey during and after the event. We've got 39 responses in total.
+
+- **Have you been satisfied with PyCon CZ 2017?** 94.9% yes!
+- **I came to PyCon CZ to/because...**
+    - 53.8% - networking and meeting people
+    - 51.3% - talks and learning about Python
+- **Which talk you enjoy the most?**
+    1. 12.8% - [Naomi Ceder](https://twitter.com/NaomiCeder) ([What lies ahead? Python’s future strengths and challenges](https://cz.pycon.org/2017/speakers/detail/talk/34/)) and [Lilly Ryan](https://twitter.com/attacus_au) (any of the [two](https://cz.pycon.org/2017/speakers/detail/talk/9/) [talks](https://cz.pycon.org/2017/speakers/detail/talk/31/))
+    2. 10.3% - [Lumír Balhar](https://twitter.com/lumirbalhar) & [Petr Viktorin](https://twitter.com/NaomiCeder) ([How we started teaching Python](https://cz.pycon.org/2017/speakers/detail/talk/14/))
+    3. 7.7% - Kateřina Falk ([Science with the world’s biggest lasers](https://cz.pycon.org/2017/speakers/detail/talk/35/))
+- **What were the top 3 things you most enjoyed at the event this year?**
+    - 41.0% - community / people
+    - 35.9% - venue / building / location
+    - 25.6% - talks
+    - 25.6% - atmosphere
+    - 15.4% - coffee
+- **What were the "bottom 3" things we could have done better?**
+    - 20.5% - food
+    - 20.5% - Wi-Fi
+    - 12.8% - rooms, chairs
+    - 12.8% - afterparty
+    - 7.7% - venue / building / location
+- **Where did you travel from?**
+    - 38.5% - Prague, Czech Republic
+    - 7.7% - Brno, Czech Republic
+    - 7.7% - Poland
+    - 5.1% - not far from Prague, Czech Republic
+    - 5.1% - Bratislava, Slovakia
+    - 5.1% - Czech Republic
```

Comment:
tady ten poslední řádek 5.1% CZ bych škrtla

Reply:
No, to nejde. Vypisoval jsem to do určitého procenta, takže by mi dávalo smysl škrtnout všechny tři, které mají 5.1%, nebo všechny tři ponechat. Zde je Czech Republic ve smyslu "far from Prague, Czech Republic". Možná by se to mělo napsat přímo takto, aby to bylo jasnější. I když "far from" není přesné. Je to spíš "not not far from Prague" 😄 Tedy asi něco jako "further from Prague, Czech Republic"

Reply:
a taky ne Brno, ne? :)
Co třeba „others from Czech Republic“?